includes/admin/views/tmpl-order-item.php

```diff
@@ -20,8 +20,9 @@
 <td class="has-row-actions column-primary">
 	<div class="removable">
 		<# if ( true === data.state.isAdding ) { #>
-		<button class="button-link delete">
-			<span class="dashicons dashicons-no"></span>
+		<label for="remove-item" class="screen-reader-text">__( Remove Item )</label>
```

Comment:
The text string can be in just a span after the dashicons one--the `label` is not needed for a `button`, which can accept text (just in this case, it won't be visible because of the screen-reader-text class).

Also, you'll need to use inline PHP to use the translation function with the text domain, and echo, so the whole span may end up looking something like this (not tested):

```php
<span class="screen-reader-text"><?php esc_html_e( 'Remove Item', 'easy-digital-downloads' ); ?></span>
```